lib/unleash/bootstrap.rb

```diff
@@ -0,0 +1,55 @@
+require 'unleash/configuration'
+require 'unleash/feature_toggle'
+require 'logger'
+require 'time'
+require 'net/http'
+require 'uri'
+
+module Unleash
+  class FileBootStrapper
+    attr_accessor :file_path
+
+    def initialize(file_path)
+      self.file_path = file_path
+    end
+
+    def read
+      file_content = File.read(self.file_path)
+      bootstrap_hash = JSON.parse(file_content)
+      Unleash.extract_bootstrap(bootstrap_hash)
+    end
+  end
+
+  class UrlBootStrapper
+    attr_accessor :uri, :headers
+
+    def initialize(uri, headers)
+      self.uri = URI(uri)
+      self.headers = headers
+    end
+
+    def read
+      request = Net::HTTP::Get.new(self.uri, self.build_headers(self.headers))
```

Comment:
Note that as a "Built-in" class, then it should use the functionalities from Unleash::Util::Http

Reply:
Fixed after the merge of #86